src/platform/testing/unit/axe-plugin.js

```diff
@@ -0,0 +1,50 @@
+import axe from 'axe-core';
+
+module.exports = function(chai, utils) {
+  const Assertion = chai.Assertion;
+
+  utils.addMethod(chai.Assertion.prototype, 'accessible', function(
+    rules = {},
+    rulesets = ['section508', 'wcag2a', 'wcag2aa', 'best-practice'],
+  ) {
+    const el = this._obj;
+    const config = {
+      runOnly: {
+        type: 'tag',
+        values: rulesets,
+      },
+      rules: {
+        region: { enabled: false },
```

Comment:
I think we have to disable this one because our JSDom setup doesn't put elements inside a landmark.

Reply:
Completely valid. JSDom sets up the barest of HTML wrappers. We had to do something similar for fragment unit tests for Liquid templates.